CMakeLists.txt

```diff
@@ -44,7 +44,11 @@ common_find_package_post()
 
 set(ROCKETS_DEPENDENT_LIBRARIES Threads)
 
-add_subdirectory(apps)
+option(ROCKETS_BUILD_APPLICATIONS "Build example applications" ON)
```

Comment:
I prefer ON by default, otherwise they will almost never be built by CI and people may forget about them altogether when doing dev if they have to enable them manually. I think docker is the special case where we want to minimize size, but even then normally the apps should not be build because CMake build only builds required targets, no?

Reply:
Not with PROJECT-install. all targets from all dependencies are considered there. The install might not be necessary in the container altogether (now that I'm thinking about it), but it's ok probably to keep it enabled from an CI point of view. On the hand, we have CI specific (TRAVIS=1) handling also in other places, where we could enable all the optional stuff to build it.